config_defaults.yml

```diff
@@ -29,13 +29,25 @@
     env: PUPPET_GEM_VERSION="~> 3.0"
     bundler_args: --without system_tests
 Gemfile:
+  supports_windows:
+    - puppetlabs-acl
+    - puppetlabs-chocolatey
+    - puppetlabs-dism
+    - puppetlabs-dsc
+    - puppetlabs-powershell
+    - puppetlabs-reboot
+    - puppetlabs-registry
+    - puppetlabs-sqlserver
+    - puppetlabs-wsus_client
```

Comment:
@ferventcoder `@configs` does not expose variables outside of the `config_defaults.yml` structure under the `Gemfile` key to the template as far as I saw, so this was the only location without making the template do `File.read`s, which would be worse...